modules/administration-guide/pages/configuring-authorization-for-private-git-repositories-on-bitbucket.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+:description: Configuring authorization for private Git repositories on Bitbucket
+:keywords: configuring-authorization, configure-authorization, private-repository, private-git-repository, private-repo, private-git-repo, bitbucket
+:navtitle: Configuring authorization for private Git repositories on Bitbucket
+// :page-aliases:
+
+:parent-context-of-configuring-authorization-for-private-git-repositories-on-bitbucket: {context}
+
+[id="configuring-authorization-for-private-git-repositories-on-bitbucket_{context}"]
+= Configuring authorization for private Git repositories on Bitbucket
+
+To enable Che users to access a private Git repository that is hosted on a Bitbucket server:
+
+. Set up the Bitbucket application link
+. Apply the Bitbucket application link Secret
```

Comment:
```suggestion
. Apply the Bitbucket application link Secret on the {prod-short} namespace
```

Just a minor suggestion

Reply:
@dkwon17, question: What do you think, do I need to mention `{prod-short} namespace` here in the overview if I have it mentioned in the secret application procedure?
![image](https://user-images.githubusercontent.com/78564304/161536531-f036054a-8a96-4afd-8d24-fdb7f11df90f.png)

Reply:
@max-cx, mentioning {prod-short} namespace here is not absolutely necessary IMO, I suggested it to provide more clarity on the title as to where the Secret is applied.